packages/components/src/utils/data/index.ts

```diff
@@ -1,23 +1,33 @@
-import { Manifest } from '@rsdoctor/types';
+import { Constants, Manifest } from '@rsdoctor/types';
 import { useEffect, useState } from 'react';
 import { LocalServerDataLoader } from './local';
 import { RemoteDataLoader } from './remote';
 import { BaseDataLoader } from './base';
 import { getAPILoaderModeFromStorage } from '../storage';
 import { APILoaderMode4Dev } from '../../constants';
+import { BriefDataLoader } from './brief';
 
-const loaderTask = new WeakMap<Manifest.RsdoctorManifestWithShardingFiles, Promise<BaseDataLoader>>();
+const loaderTask = new WeakMap<
+  Manifest.RsdoctorManifestWithShardingFiles,
+  Promise<BaseDataLoader>
+>();
 
-async function createDataLoader(manifest: Manifest.RsdoctorManifestWithShardingFiles) {
+async function createDataLoader(
+  manifest: Manifest.RsdoctorManifestWithShardingFiles,
+) {
   try {
     if (process.env.NODE_ENV === 'development') {
       const mode = getAPILoaderModeFromStorage();
-      if (mode === APILoaderMode4Dev.Local) return new LocalServerDataLoader(manifest);
-      if (mode === APILoaderMode4Dev.Remote) return new RemoteDataLoader(manifest);
+      console.log(`[development]-${mode}`);
+      if (mode === APILoaderMode4Dev.Local)
+        return new LocalServerDataLoader(manifest);
+      if (mode === APILoaderMode4Dev.Remote)
+        return new RemoteDataLoader(manifest);
     }
 
     // local server exists
     if (manifest.__LOCAL__SERVER__) {
+      console.log(`[LocalServerDataLoader]`);
```

Comment:
why not use debug console

Reply:
This is client log, do not need debug env mode.